package/aws/README.md

```diff
@@ -49,6 +49,27 @@ The process of creating an RBAC Wrapper for an AWS resource is as follows:
 The resource folder must sit in the package//aws folder. The folder name must match the AWS resource name. 
 For example: `package/aws/bucket` for the `bucket` resource from provider-aws.
 
+### Naming conventions
+
+We use the following naming conventions in our RBAC Wrapper compositions.
+
+#### Namespace
+
+Our namespace must be [group].crossplane.dfds.cloud. [group] should be a generic descriptive category such as:
+
+- storage
+- networking
+- compute
+- messaging
+
+#### XRD
+
+Our Composite Resource Definitions should follow the naming of X[cloud][resource-name-defined-by-crossplane]. X stands for Composite. For example, if the resource is an AWS Bucket then the name of XAWSBucket. If the resource is an Azure Blob Storage Container then the name would be XAzureContainer. Apply the plural resource name where appropriate.

```

Comment:
X[vendor-name][resource-name-defined-by-crossplane] ? since a vendor does not always mean cloud :)

Reply:
Good suggestion. Amended